docs/vs-tools-tizen/tools/profiler-user-manual.md

```diff
@@ -1,137 +1,154 @@
-
-   * To clear the selected interval, click any place in the chart. Data in the right panel will be reset as well.
-   * Methods Panel consists 3 tabs: Call Tree, Hot Path, Hot Lines
-      1) Call Tree Tab displays Top Methods & Call Tree on the selected time interval for the selected thread
-       
-         ![Call Tree Tab](../image/call_tree_tab.png)
-
-      2) Hot Path Tab displays Hot Path on the selected time interval for the selected thread
-      
-         ![Hotpath Tab](../image/hotpath_tab.png)
-
-         1) In the collapsed state, the Hot Path Item displays the hottest method. When expanding – the whole path from this method will the root calling method is displayed.
-
-      3) Hot Lines Tab displays Top Source Lines on the selected time interval for the selected thread (source lines numbers are displayed in brackets […])
-       
-         ![Hotline Tab](../image/hotline_tab.png)
-         
-   * Depending on the profiling mode chosen at the left, different data is displayed. Table 1 shows different types of data displayed in the right panel. Filtered by Time – shows how much time every method has occupied (highlighted in red). Filtered by Allocated memory – shows an amount of memory allocations per method (in Kb or Mb). Filtered by Samples – shows amount of samples per method.
-    
-     ![Profiling Categories](../image/profiling_categories.png)
-   * Double click on the selected line in Hot Lines Tab opens a source file where hot lines are highlighted with intensity that corresponds to the “hottest” level of this line
-    
-     ![Jump To Line](../image/jump_to_line.png)     
+﻿# Profiler
+
+The profiler allows you to analyze the performance of your application. You can record, manage, and view profiling sessions.
+
+## Recording Profiling Sessions
+
+To record a profiling session:
+1. In the Visual Studio menu, select **Tools &gt; Tizen &gt; Profiler &gt; Run Profiler**.  
+  The profiler configuration dialog opens.
+
+2. Define the profiling options.  
+    ![Profiler configuration](media/profiler-config.png)
+    - **Path to application** is automatically filled with the information for the current solution.
+    - **Profiling Preset**  
+    You can select from the following predefined profiling configurations:
+      - **CPU Sampling**  
+        Trace CPU and source lines, but not memory allocation.
+      - **Memory Allocation**  
+        Trace memory allocation and source lines, but not CPU.
+      - **Complex Profiling**  
+        Trace CPU, source lines, and memory allocation.
+    - **Sampling Options**  
+    You can set the following sampling options:
+      - **Sampling Interval**  
+        You can set the sampling time interval, in milliseconds. The default is 10 ms.
+      - **High Granularity**  
+        You can switch on high-granularity sampling. It enables better precision and sample regularity, at the cost of efficiency.
+    - **Trace Options**  
```

Comment:
The "Trace Garbage Collection" option visible in the screenshot is missing a description in this section.